Source/diablo.cpp

```diff
@@ -996,7 +996,7 @@ void diablo_hotkey_msg(int dwMsg)
 			app_fatal("Can't get program name");
 		separator = strrchr(Filename, '\\');
 		if (separator)
-			*separator = NULL;
+			*separator = 0;

```

Comment:
```suggestion
			*separator = '\0';
```

In general, I'd prefer using `0` for ints and `'\0'` for characters.